infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationClientRegisterListener.java

```diff
@@ -71,10 +71,10 @@ public LogReplicationClientRegisterListener(SessionManager sessionManager,
      */
     public void start() {
         CorfuStoreMetadata.Timestamp timestamp = configManager.preprocessAndGetTail();
-
-        log.info("Start log replication listener for client registration table from {}", timestamp);
+        CorfuStoreMetadata.Timestamp clientSubscriptionTimeStamp = CorfuStoreMetadata.Timestamp.newBuilder().setEpoch(timestamp.getEpoch()).setSequence(timestamp.getSequence()-1).build();
```

Comment:
I haven't found a better way to address this. When I was subscribing the log tail, the code path skipped onNext(), hence skipped the whole workflow of custom client registration, and the ITs in LogReplicationRoutingQueueIT failed. After some debugging, I found out that if we subscribe t, 0 <= t <= log tail - 1, the code path won't skip onNext(). So I subscribe to the sequence number that is closest to the log tail.

Reply:
I found this comment from one of the subscribe method in StreamingManager, "* @param lastAddress      last processed address, new notifications start from lastAddress + 1”,
so if we subscribe exactly the log tail, we will only get the notifications from logtail + 1, which skips the update at logtail? If this is true, it can be used to explain why we need to subscibe a timestamp that is less than logtail.

Reply:
nit: incorrect indentation